From e342535cdfd1aa36422262f0e2dda754e11e361f Mon Sep 17 00:00:00 2001
From: Guenter Roeck <linux@roeck-us.net>
Date: Sun, 15 Jan 2017 23:10:32 -0800
Subject: [PATCH 3/5] drivers/input/misc/tps65218-pwrbutton.c fixup

Signed-off-by: Guenter Roeck <linux@roeck-us.net>
---
 drivers/input/misc/tps65218-pwrbutton.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/input/misc/tps65218-pwrbutton.c b/drivers/input/misc/tps65218-pwrbutton.c
index 8757d2690c40..33f48a622fc6 100644
--- a/drivers/input/misc/tps65218-pwrbutton.c
+++ b/drivers/input/misc/tps65218-pwrbutton.c
@@ -129,7 +129,9 @@ static int tps6521x_pb_probe(struct platform_device *pdev)
 	}
 
 	error = devm_request_threaded_irq(dev, irq, NULL, tps6521x_pb_irq,
-					  IRQF_TRIGGER_RISING | IRQF_TRIGGER_FALLING | IRQF_ONESHOT,
+					  IRQF_TRIGGER_RISING |
+						IRQF_TRIGGER_FALLING |
+						IRQF_ONESHOT,
 					  pwr->data->name, pwr);
 	if (error) {
 		dev_err(dev, "failed to request IRQ #%d: %d\n", irq, error);
-- 
2.7.4

