From b43bf730883900ba438197796cc5a4159871f4f9 Mon Sep 17 00:00:00 2001
From: Guenter Roeck <linux@roeck-us.net>
Date: Sun, 15 Jan 2017 23:10:15 -0800
Subject: [PATCH 2/5] drivers/input/misc/adxl34x-i2c.c fixup

Signed-off-by: Guenter Roeck <linux@roeck-us.net>
---
 drivers/input/misc/adxl34x-i2c.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/input/misc/adxl34x-i2c.c b/drivers/input/misc/adxl34x-i2c.c
index 3d1c1e56c77c..fd1f0af27cbb 100644
--- a/drivers/input/misc/adxl34x-i2c.c
+++ b/drivers/input/misc/adxl34x-i2c.c
@@ -89,7 +89,9 @@ static int adxl34x_i2c_probe(struct i2c_client *client,
 	}
 
 	ac = adxl34x_probe(dev, client->irq, false,
-			   i2c_check_functionality(client->adapter, I2C_FUNC_SMBUS_READ_I2C_BLOCK) ? &adxl34x_smbus_bops : &adxl34x_i2c_bops);
+			   i2c_check_functionality(client->adapter,
+						   I2C_FUNC_SMBUS_READ_I2C_BLOCK) ?
+				&adxl34x_smbus_bops : &adxl34x_i2c_bops);
 	if (IS_ERR(ac))
 		return PTR_ERR(ac);
 
-- 
2.7.4

