From 990ab1055222ff7a12ca93e237ee8533a4f541de Mon Sep 17 00:00:00 2001
From: Guenter Roeck <linux@roeck-us.net>
Date: Sun, 15 Jan 2017 23:11:02 -0800
Subject: [PATCH 5/5] drivers/input/touchscreen/ad7879-spi.c fixup

Signed-off-by: Guenter Roeck <linux@roeck-us.net>
---
 drivers/input/touchscreen/ad7879-spi.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/input/touchscreen/ad7879-spi.c b/drivers/input/touchscreen/ad7879-spi.c
index f85c6c0ff6e4..7c324ac499be 100644
--- a/drivers/input/touchscreen/ad7879-spi.c
+++ b/drivers/input/touchscreen/ad7879-spi.c
@@ -126,8 +126,8 @@ static int ad7879_spi_probe(struct spi_device *spi)
 	spi->bits_per_word = 16;
 	err = spi_setup(spi);
 	if (err) {
-	        dev_dbg(dev, "spi master doesn't support 16 bits/word\n");
-	        return err;
+		dev_dbg(dev, "spi master doesn't support 16 bits/word\n");
+		return err;
 	}
 
 	ts = ad7879_probe(dev, AD7879_DEVID, spi->irq, &ad7879_spi_bus_ops);
-- 
2.7.4

